### PR TITLE
[UIPO-40] Product order component

### PR DESCRIPTION
**Pull request by mirafedas**
Please check this [commit](https://github.com/Nazar-bnm/ui-internship/pull/33/commits/8cdae23bfe7ab790e84123cab6cc665d41e52180) - here are changes related to the Product Order Component.

### CODE REVIEW

File: src/components/ProductOrder/ProductOrder.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Dropdown from '../Dropdown';
+import Button from '../Button';
+import './ProductOrder.scss';
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.js:6]: put an empty line after this one

File: src/components/ProductOrder/ProductOrder.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    { value: '3', label: '3' },
+    { value: '4', label: '4' },
+    { value: '5', label: '5' },
+    { value: '6', label: '6' },
+    { value: '7', label: '7' },
+    { value: '8', label: '8' },
+    { value: '9', label: '9' },
+    { value: '10', label: '10' },
+
+  ],
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.js]: Should be imported from ProductOrderMockups.js. Don't use such an approach in component directly

File: src/components/ProductOrder/ProductOrder.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    </div>);
+};
+
+ProductOrder.propTypes = {
+  itemName: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  price: PropTypes.string.isRequired,
+  quantity: PropTypes.array.isRequired,
+  sizes: PropTypes.array.isRequired,
+  colors: PropTypes.array.isRequired,
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.js]: By the way, using isRequired means you absolutely will not use defaultProps

File: src/components/ProductOrder/ProductOrder.spec.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  beforeEach(() => {
+    productOrderParameters;
+
+    wrapper = shallow(
+      <ProductOrder {...props} />
+    );
+  });
+  test('should match snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.spec.js]: 🤔If you have only 1 test-case consider putting everything in it instead

File: src/components/ProductOrder/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import ProductOrder from './ProductOrder';
+
+export default ProductOrder;
```
Comment (Hollister009) [inline src/components/ProductOrder/index.js]: `export {default} from './ProductOrder'`

**Review by Hollister009 — COMMENTED** [review]
Minor fixes and you are good to go!

File: src/components/ProductOrder/ProductOrder.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+ProductOrder.propTypes = {
+  itemName: PropTypes.string,
+  description: PropTypes.string,
+  price: PropTypes.string,
+  quantity: PropTypes.array,
+  sizes: PropTypes.array,
+  colors: PropTypes.array,
+};
+
+ProductOrder.defaultProps = { ...productOrderParameters };
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.js]: It's not what I exactly meant 😅

Comment (mirafedas) [reply src/components/ProductOrder/ProductOrder.js]: Could you please explain a bit more? :)

Comment (Hollister009) [reply src/components/ProductOrder/ProductOrder.js]:
```
ProductOrder.defaultProps = {
  itemName: '',
  description: '',
  price: '',
  quantity: [],
  sizes: [],
  colors: [],
};
```

Comment (Hollister009) [reply src/components/ProductOrder/ProductOrder.js]: quantity - should be array?

Comment (Hollister009) [reply src/components/ProductOrder/ProductOrder.js]: It's like you declaring properties with empty values by default

File: src/components/ProductOrder/ProductOrder.spec.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+import { productOrderParameters } from '../../config/ProductOrderMockups';
+
+describe('<ProductOrder />', () => {
+  const wrapper = shallow(
+    <ProductOrder {...productOrderParameters} />
+  );
+
+  test('should match snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
```
Comment (Hollister009) [inline src/components/ProductOrder/ProductOrder.spec.js]:
```
describe('<ProductOrder />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(
      <ProductOrder {...productOrderParameters} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
```

**Review by Nazar-bnm — APPROVED** [review]

**Pull request merged**